manifest.yaml

```diff
@@ -8,7 +8,7 @@ containers:
         mountPath: /data
 
   - name: guestbook
-    image: google/guestbook-python-redis
+    image: gcr.io/your-project-id/guestbook-python-redis
```

Comment:
PTAL, I basically reverted the manifest, and changed the wording at the bottom of the README, which talks about modifying the app.